scripts/apps/search/views/saved-search-manage-subscribers.html

```diff
@@ -0,0 +1,183 @@
+<div sd-modal class="modal" data-model="wrapper.modalOpen" on-modal-close="onInternalModalClose">
+    <div class="modal__header">
+        <a href="" class="close" ng-click="manageSubscriptions(false)"><i class="icon-close-small"></i></a>
+        <h3 class="modal__heading" translate="">
+            <span translate>Manage: </span>
+            <strong>{{savedSearch.name}}</strong>
+        </h3>
+    </div>
+
+    <div class="modal__body">
+        <!-- if start -->
+        <div ng-if="wrapper.subscriptionInCreateOrEditMode == null">
+            <div class="space-between" style="padding-bottom: 2rem; color: #898989">
+                <div>
+                    <strong>{{getSubscriptionCount()}}</strong>
+                    <span translate>subscriptionss</span>
```

Comment:
typo `s/ss</s</`

Reply:
I'm not sure I understand your formatting :D - `s/ss</s</`. I see that there's an extra `s` in `<span translate>subscriptionss</span>`. Do you see more typos in this file?